### ISSUE TITLE
Ar cloud anchor

### ISSUE THREAD

**Issue by dev19872014** [google-ar/arcore-android-sdk#373]
If auser has created AR experience at one place then closed the app & reopen it later, then is it possible that user will find it at the same place?

**Comment by sam598** [comment]
If you store the ID of the anchor, you can attempt to resolve it for up to 24 hours.

**Comment by inio** [comment]
Yes, as @sam598 said, cloud anchors may be resolved for up to 24 hours after they are initially hosted.
